inventory.schema.json

```
@@ -247,6 +247,9 @@
             },
             "enclosureserial": {
               "type": "string"
+            },
+            "secure_boot_enabled": {
+              "type": "boolean"
```

We probably should better support the following status: "enabled", "disabled" (involving it is supported), "unsupported"
So may I suggest to use this:
```suggestion
            "secure_boot": {
              "type": "string",
              "pattern": "^(enabled|disabled|unsupported)$"
```